assets/js/googlesitekit-idea-hub-notice.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Idea Hub Notice (for Block Editor) entrypoint.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { getItem, setItem } from './googlesitekit/api/cache';
+
+const shownNotices = [];
```

Comment:
This variable belongs to the `loadIdeaHubNotices` function context. Let's move it into that function then.